packages/apps/src/SideBar/Item.tsx

```diff
@@ -15,27 +15,61 @@ import accountObservable from '@polkadot/ui-keyring/observable/accounts';
 import { withApi, withMulti, withObservable } from '@polkadot/ui-api/index';
 import { isFunction } from '@polkadot/util';
 
+import ReactTooltip from 'react-tooltip';
+
 type Props = I18nProps & ApiProps & {
+  isCollapsed: boolean,
   allAccounts?: SubjectInfo,
   route: Route
 };
 
+interface Tooltip {
+  'data-tip': boolean;
+  'data-for': string;
+  'data-tip-disable'?: boolean;
+}
+
 class Item extends React.PureComponent<Props> {
+
+  componentWillUpdate () {
+    ReactTooltip.rebuild();
+  }
   render () {
-    const { route: { i18n, icon, name }, t } = this.props;
+
+    const { route: { i18n, icon, name }, t, isCollapsed } = this.props;
 
     if (!this.isVisible()) {
       return null;
     }
 
+    let tooltip: Tooltip = {
+      'data-tip': true,
+      'data-for': `nav-${name}`
+    };
+    isCollapsed
```

Comment:
Can just do

```js
const tooltip: Tooltip = {
  'data-for': `nav-${name}`,
  'data-tip': true,
  'data-tip-disable': isCollapsed
};
```

Couple of this here -

- `const`
- no need to create `?`` outside the structure
- alphabetical ordering